fiber/client.py

```diff
@@ -62,3 +62,23 @@ def subscribe_new_txs(self):
 
     def subscribe_new_blocks(self):
         return self.stub.SubscribeNewBlocks(api_pb2.google_dot_protobuf_dot_empty__pb2.Empty(), metadata=self.metadata)
+    
+    def send_transaction(self, tx: Transaction):
+        res = self.stub.SendTransaction(tx, metadata=self.metadata)
+        
+        return res.hash, res.timestamp
```

Comment:
The gRPC call takes `api_bp2.Transaction` as its type, so we should add a `tx_to_proto` method for converting our `Transaction` type to that before sending it off